lib/checks/visibility/hidden-content.js

```diff
@@ -1,13 +1,16 @@
-let styles = window.getComputedStyle(node);
+const whitelist = ['SCRIPT', 'HEAD', 'TITLE', 'NOSCRIPT', 'STYLE', 'TEMPLATE'];
+if (
```

Comment:
I agree that one line is more readable, as long as it doesn't run over 80-100 chars.